src/main/kotlin/carGame/domain/CarName.kt

```diff
@@ -3,6 +3,10 @@ package carGame.domain
 @JvmInline
 value class CarName(val value: String) {
     init {
-        require(value.length <= 5) { "자동차 이름은 5자를 초과할 수 없습니다." }
+        require(value.length <= MAX_NAME_LENGTH) { "자동차 이름은 ${MAX_NAME_LENGTH}자를 초과할 수 없습니다." }
     }
-}
\ No newline at end of file
+
+    companion object {
+        const val MAX_NAME_LENGTH = 5
```

Comment:
```suggestion
        private const val MAX_NAME_LENGTH = 5
```
클래스 내부에서만 접근한다면 private 접근자를 붙여보면 어떨까요?